packages/server/test/middleware/nuxt.test.js

```diff
@@ -306,4 +306,20 @@ describe('server: nuxtMiddleware', () => {
     expect(await nuxtMiddleware(req, res, next)).toBe(err)
     expect(consola.error).toBeCalledWith(err)
   })
+
+  test('should return 400 if request is uri error', async () => {
+    const context = createContext()
+    const err = Error('render error')
+    err.name = 'URIError'
+    err.statusCode = 400
+    context.renderRoute.mockImplementation((url, ctx) => {
+      throw err
+    })
+    const nuxtMiddleware = createNuxtMiddleware(context)
+    const { req, res, next } = createServerContext()
```

Comment:
Can we somehow corrept req.url here instead of mock and error?

Reply:
I made inappropriate req.url as below.
```
    await nuxtMiddleware({ ...req, url: 'http://localhost/test/server/%c1%81' }, res, next)
```